solver/src/http_solver/model.rs

```diff
@@ -67,4 +74,11 @@ pub struct UpdatedUniswapModel {
     pub balance_update1: i128,
     #[serde(with = "serde_with::rust::display_fromstr")]
     pub balance_update2: i128,
+    pub exec_plan: ExecPlanCoordsModel,
+}
+
+#[derive(Debug, Deserialize)]
+pub struct ExecPlanCoordsModel {
```

Comment:
nit: I'm not a fan of abbreviations (they save you a little time to type but take more time to comprehend later on). Can we call this ExecutionPlanModel or ExecutionPlanCoordinates or ExecutionPlanCoordinatesModel?